code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm

```diff
@@ -2310,6 +2334,21 @@
 	..()
 	M.apply_effect(max(M.total_radiation - 1 * removed, 0), IRRADIATE, blocked = 0)
 
+/datum/reagent/alcohol/ethanol/mushroomvodka
```

Comment:
This can just be a subtype of the regular vodka reagent, to avoid copypasta of the affect_ingest

Reply:
All done.